dags/incline_dash.py

```diff
@@ -37,10 +37,11 @@
 
     migrated_clients = bigquery_etl_query(
         task_id="generate_migrated_clients",
-        destination_table=table,
+        destination_table=None,
         project_id=project,
         dataset_id=dataset,
         date_partition_parameter=None,
+        parameters=('--nouse_legacy_sql'),
```

Comment:
This should already be handled by the `.bigqueryrc` installed in the docker image.

Reply:
Ah, that's why it didn't work locally.